src/integration-tests/commonTests.js

```diff
@@ -58,14 +64,85 @@ const logIn = async () => {
 	}
 }
 
+const createUserRoles = async () => {
+	let request = defaults(supertest('http://localhost:5001'))
+	await waitForService(baseURL)
+	jest.setTimeout(10000)
+
+	let email = 'orgadmin' + crypto.randomBytes(5).toString('hex') + '@shikshalokam.com'
+	let password = 'Welcome@123'
+	let res = await request.post('/user/v1/account/create').send({
+		name: 'orgadmin',
+		email: email,
+		password: password,
+	})
+
+	if (res.body && res.body.result && res.body.result.access_token && res.body.result.user.id) {
+
+		defaultHeaders = {
+			'X-auth-token': 'bearer ' + res.body.result.access_token,
+			Connection: 'keep-alive',
+			'Content-Type': 'application/json',
+		}
+
+		let existingCreatorRole = await request.get('/user/v1/user-role/list')
+			.set(defaultHeaders)
+			.query({
+				title: 'content_creator',
+				organization_id: 1,
+			})
+
+		if (existingCreatorRole.body.result?.data?.length == 0) {
+			let createCreatorRole = await request.post('/user/v1/user-role/create')
+				.set(defaultHeaders)
+				.send({
+					title: 'content_creator',
+					user_type: 0,
+					organization_id: 1,
+					label: 'Content Creator',
+					visibility: 'PUBLIC'
+				})
+
+			if (createCreatorRole.statusCode != 200) {
+				console.log('Role Creation Failed')
```

Comment:
can we give specific error message or the role title / code to better understand?  @priyanka-TL